### PR TITLE
Minimalistic theme

### PR DESCRIPTION
**Pull request by frankystone**
Hopefully i did all correct now.

It would be good if `loadscreens/gameloading/` uses also the default images if this folder is missing. Or convert `*.png` with `*.jpg` for this big images.

### CODE REVIEW

File: addons/minimalistic_theme.wad/addon [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Automatically created by Widelands 1.0~git25202[9f820f0@master] (Debug)
+
+[global]
+name=_"Minimalistic Theme"
+description=_"Minimalistic theme whith reduced colors. Note that you have to enable the theme in the options menu after installing."
+author=_"kaputtnik"
```
Comment (Noordfrees) [inline addons/minimalistic_theme.wad/addon:6]:
```suggestion
description=_"Minimalistic theme with reduced colors. Note that you have to enable the theme in the options menu after installing."
author="kaputtnik"
```

**Review by Noordfrees — CHANGES_REQUESTED** [review]
I'm unable to download this add-on from your branch from the add-ons manager. Perhaps you forgot to re-run `java UpdateList` after some change?
```
[00:00:27.778 real] ERROR: download_addon minimalistic_theme.wad: [../src/network/net_addons.cc:245] Downloaded file 'temp/minimalistic_theme.wad.tmp/addon': Unable to check output file: [../src/network/net_addons.cc:241] Downloaded file 'temp/minimalistic_theme.wad.tmp/addon': Checksum mismatch, found 559beb0d5859a47b6ea03a9d7cc03436, expected 50eb22769143cee25aada3ab2c9d765c
```

**Comment by Noordfrees** [comment]
> It would be good if `loadscreens/gameloading/` uses also the default images if this folder is missing. Or convert `*.png` with `*.jpg` for this big images.

It is safe to use JPG instead of PNG. The reason why these photographs were converted to PNG in the default theme is that a reviewer complained they were getting painfully fuzzy at certain resolutions and requested using PNG because it scales in a more eye-friendly way.

**Comment by frankystone** [comment]
Thanks for the suggestions :-)

I'll do some clean in `init.lua`, so this is a draft now.

**Comment by Noordfrees** [comment]
The add-on downloads fine now. The remote shows up as having 0 screenshots though because the screenie folder name is misspelled as "minimalisic" instead of "minimalistic".

**Comment by frankystone** [comment]
Ready now, hopefully :-)

**Review by Noordfrees — APPROVED** [review]

**Pull request merged**